en/developer_guide/multiPanel.rst

```diff
@@ -0,0 +1,83 @@
+Add panel to an administration view
+===================================
+
+Context
+-------
+
+When working with administration back office, sometimes you need to add your own forms or contents to the existing ones.
```

Comment:
I find this introduction a little confusing. I don't see the aim of the doc. Maybe you could rephrase it.